apps/workers/crawlerWorker.ts

```diff
@@ -5,9 +5,11 @@ import type { Job } from "bullmq";
 import type { Browser } from "puppeteer";
 import { Readability } from "@mozilla/readability";
 import { Mutex } from "async-mutex";
+import Database from "better-sqlite3";
```

Comment:
I don't think we should leak `better-sqlite3` here. I'll re-export the type that we need from `packages/db`.